src/administration/install_docker.md

```diff
@@ -4,7 +4,7 @@ Make sure you have both docker and docker-compose(>=`1.24.0`) installed. On Ubun
 
 ```bash
 # Create a folder for the lemmy files. the location doesnt matter, you can put this anywhere you want
-mkdir /lemmy
+mkdir lemmy
 cd /lemmy
```

Comment:
Need to change this line also.

Reply:
Oh oops - you mean line 8 right? fixed it